easybuild/easyblocks/t/totalview.py

```diff
@@ -28,13 +32,32 @@ class EB_TotalView(EasyBlock):
     def __init__(self, *args, **kwargs):
         """Initialisation of custom class variables for Totalview"""
         super(EB_TotalView, self).__init__(*args, **kwargs)
-        if not self.cfg['license_file']:
-            self.cfg['license_file'] = 'UNKNOWN'
+        
+        self.license_var = 'UNKNOWN'
+        self.license_env_var = 'UNKNOWN'
 
     def configure_step(self):
-        """No configuration for TotalView."""
-        if not os.path.exists(self.cfg['license_file']):
-            raise EasyBuildError("Non-existing license file specified: %s", self.cfg['license_file'])
+        """
+        Handle of license
+        """
+        default_lic_env_var = 'LM_LICENSE_FILE'
+        lic_specs, self.license_env_var = find_flexlm_license(custom_env_vars=[default_lic_env_var],
+                                                              lic_specs=[self.cfg['license_file']])
+
+        if lic_specs:
+            if self.license_env_var is None:
+                self.log.info("Using Totalview license specifications from 'license_file': %s", lic_specs)
+                self.license_env_var = default_lic_env_var
+            else:
+                self.log.info("Using Totalview license specifications from %s: %s", self.license_env_var, lic_specs)
+
+            self.license_file = os.pathsep.join(lic_specs)
+            self.cfg['license_file'] = self.license_file
```

Comment:
there's no point in updating this, since it's not used later (only `self.license_file` is)

besides, it's better not to fiddle with the parsed easyconfig